src/commands/rsync.ts

```diff
@@ -20,10 +42,29 @@ export const rsyncCommand = async () => {
         placeHolder: "Which plugin would you like to sync?",
     });
 
-    const wpPath = await vscode.window.showInputBox({
-        prompt: 'Enter the remote path to upload the plugin contents to.',
-        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
-    });
+	const existingSite = await vscode.window.showQuickPick(getSavedSites(configFilePath), {
+		placeHolder: "Which site would you like to sync to?"
+	});
+
+	if (typeof existingSite === 'undefined') {
+		console.log('No site selected, exiting.');
+		return;
+	}
+
+	let wpPath: string | undefined;
```

Comment:
Nit: Consider using a ternary to keep things clean and concise :)

```
const wpPath = existingSite === '+ Add New Site'
    ? await vscode.window.showInputBox({ ... })
    : getSavedPath(existingSite);
```

Reply:
Slick! updated in bbbc1e672947aa95e7d9b7e3ab97cb989bfc05c8